src/Commands/InitCommand.php

```diff
@@ -76,6 +77,12 @@ public function handle(): void
 
         $kebabName = Str::kebab($this->appName);
 
+        $this->emailTeamLead = $this->validateInput(
+            method: fn () => $this->ask('Please specify a Code Owner/Team Lead\'s email', Str::before($this->reviewer, '@')),
```

Comment:
the `reviewer` field is always empty on this step

```suggestion
            method: fn () => $this->ask('Please specify a Code Owner/Team Lead\'s email'),
```